contributor/01-introduction-article.md

```diff
@@ -0,0 +1,97 @@
+# Draft comments
+
+ Please watch the [Intro to InnerSource Video](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504)
+* Also watch the [Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925) to the Trusted Committer role.
+* The [Lowering the Barriers to Entry](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323929) video of the Trusted Committer role can be a good addition to understand more of why the Trusted Committer might ask for different things.
+
+## Outline based Target
+### The Contributor in InnerSource
+
+  - Introduce the speaker. - only required for the video script I'd say. 
+  - Quick overview of the _Contributor_ role.
+  - Summary of the roles of InnerSource and how each relates to _Contributor_.
+  - Summary of the upcoming segments and what each covers.
+
+Outline for this section inspired by the [Trusted Committer introduction article.
+
+## Video Play-book based Target
+How do I best contribute to an InnerSource project?  This segment shares a summary of what it means to be an InnerSource Contributor.  We'll also give an overview of the upcoming training segments on being a great InnerSource Contributor.
+
+
+
+# Article
+
+## The Contributor in InnerSource.
+
+Without contributions, an InnerSource community and its project(s) would not exist in a meaningful way.
```

Comment:
This paragraph makes the case for why contributors are needed and useful, which is great.  I wonder if it would be good to change the perspective, though.  I feel like this perspective assumes that one is already committed to having a vibrant array of InnerSource communities at the company and wants to know how they can help that along.  There may be some people like that around, but I'd suggest that it's not the broadest way to appeal to potential contributors.

What about a more personal approach?  Something like:

```
Have you ever been blocked in your next coding task because another team didn't have time to add a feature in their system that you depend on?  Perhaps after a while you even had to do some extra work in your project to work around the missing feature.  How nice would it be to never be blocked in this way?

With projects that run via InnerSource, you'll never be blocked waiting for another team to deliver some needed feature.  If you're not getting what you need you can make the change you need directly in the other team's code repository by acting as an InnerSource Contributor.

...
```

Reply:
The exact words here that I'm suggesting aren't the main point as much as the persona to-whom we're talking and what we assume they value.  That's the real suggestion I'm trying to give, here.

Reply:
I agree with your point of the persona we're talking to - I guess this was a case of academic writing style coming through for a bit again. ;)
The [word suggestion](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/98/files#r312260530) you made sounds fine to me.
Could you convert that to a commit suggestion? I'll merge and then we can continue from there.

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/111

Reply:
Have a look at the updated PR.

Reply:
> Have a look at the updated PR.

Took a look, liked what I saw.